app/javascript/tests/public_health/bulk_actions/CloseRecords.test.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CloseRecords from '../../../components/public_health/bulk_actions/CloseRecords';
+import { mockPatient1, mockPatient2, mockPatient6 } from '../../mocks/mockPatients';
+import { mockMonitoringReasons } from '../../mocks/mockMonitoringReasons';
+
+const onCloseMock = jest.fn();
+const mockToken = 'testMockTokenString12345';
+
+function getWrapper(patientsArray) {
+  let wrapper = shallow(<CloseRecords authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} monitoring_reasons={mockMonitoringReasons} />);
+  return wrapper;
+}
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('CloseRecords', () => {
+  it('Sets intial state after mount correctly', () => {
```

Comment:
not sure this test is necessary because there isn't a `componentDidMount` with anything crazy happening - @DPC15038 thoughts?

Reply:
Yeah i'd agree. It's nice and good and all, but it's not really doing anything, and there's no expected behavior, so i'm okay if you remove it. I have no strong opinions one way or another